src/styles/learn.scss

```diff
@@ -181,7 +180,7 @@
       overflow: visible;
       overflow-y: scroll;
       position: fixed;
-      top: var(--nav-height);
+      top: calc(var(--nav-height) * 2);
```

Comment:
We need to place this style in the `about.scss`. There we have an extender for this class.